pkg/labels/array.go

```diff
@@ -124,18 +140,11 @@ nextLabel:
 func (ls LabelArray) Has(key string) bool {
 	// The key is submitted in the form of `source.key=value`
 	keyLabel := parseSelectLabel(key, '.')
-	if keyLabel.IsAnySource() {
-		for l := range ls {
-			if ls[l].Key == keyLabel.Key {
-				return true
-			}
-		}
-	} else {
-		for _, lsl := range ls {
-			// Note that if '=value' is part of 'key' it is ignored here
-			if lsl.Source == keyLabel.Source && lsl.Key == keyLabel.Key {
-				return true
-			}
+	for _, l := range ls {
+		// note inverted order; by definition, key is "looser" as it
+		// is coming from a selector.
```

Comment:
I'm not sure I'm following what's "inverted" here?

Reply:
I'm also not following what "key is looser" means?

Reply:
I'm trying to find a way to express in which direction source `any` is respected. Thoughts? This is just a small comment for the next viewer.

Maybe I should replace `Matches()` and `MatchesKey()` with just `Has(key Label)`, keeping the ordering consistent.

Reply:
OK, I made `Label.Has()` and `Label.HasKey()` that works the same way as `LabelArray.Has()` and `Labels.Has()`. Now everything is a lot more consistent :relieved:.